osquery/filesystem/linux/proc.h

```diff
@@ -159,10 +167,17 @@ Status procEnumerateProcesses(UserData& user_data,
       if (ret == false) {
         break;
       }
+
+      anySuccess = true;
+    } catch (const boost::filesystem::filesystem_error& e) {
+      VLOG(1) << "Exception iterating Linux processes: " << e.what()
+              << "Iterating onward\n";
```

Comment:
I recommend removing this "Iterating onward", if you feel strongly about keeping it then add ` ` (space) and remove the `\n`.

Reply:
👍